anesthetic/plot.py

```diff
@@ -510,7 +511,7 @@ def kde_contour_plot_2d(ax, data_x, data_y, *args, **kwargs):
     ymin = kwargs.pop('ymin', None)
     ymax = kwargs.pop('ymax', None)
     weights = kwargs.pop('weights', None)
-    ncompress = kwargs.pop('ncompress', 1000)
+    ncompress = kwargs.pop('ncompress', None)
```

Comment:
Why change this to `None` when docs still state `Default 1000`?

Reply:
I've pushed the alternative, which makes `MCMCSamples.plot` a little more ugly, but the lower functions more intuitive.